python/llm/src/bigdl/llm/transformers/model.py

```diff
@@ -156,16 +179,61 @@ def load_convert(cls, q_k, optimize_model, *args, **kwargs):
         # and lead to args missing.
         modules_to_not_convert = kwargs.pop("modules_to_not_convert", None)
         replace_embedding = kwargs.pop("replace_embedding", False)
+        quant_config = kwargs.pop("quantization_config", None)
         _args = copy.deepcopy(args)
         _kwargs = copy.deepcopy(kwargs)
-        try:
-            model = cls.HF_Model.from_pretrained(*args, **kwargs)
-        except NotImplementedError:
-            logger.info("Failed to load models with `low_cpu_mem_usage` specified, "
-                        "will fall to traditional load method with higher memory consumption.")
-            _kwargs["low_cpu_mem_usage"] = False
-            model = cls.HF_Model.from_pretrained(*_args, **_kwargs)
-            model.config.update({"bigdl_lcmu_enabled": False})
+        awq_config = None
+
+        if quant_config and quant_config.quant_method == "awq":
```

Comment:
is the below code copied/adapted from `transformers`? if so, add comments to show that

Reply:
It's copied from autoawq, updated.